### PR TITLE
Global layout: Larger secondary columns, adjustable box

### PR DESCRIPTION
**Pull request by andy-hook**
Fixes https://github.com/aragon/network-dashboard/issues/19

This PR adds custom column layout and adjustable box. This is needed because our UI kit has a lot of opinions about how it would like to present these, and optimises for apps with a sidebar. Our designs need a larger right hand column, and the ability to easily adjust the box padding depending on the context.

I also moved the disputable colouring into the new `LayoutBox` as a `mode` that can be optionally provided.

These changes also fix some overflow blowout we had at smaller viewport sizes.

image

### CODE REVIEW

File: src/components/Agreement/Agreement.js [hunk 1]
```diff
@@ -40,18 +42,18 @@ function AgreementLayout({ agreementDetails }) {
   } = agreementDetails
 
   return (
-    <Split
+    <LayoutColumns
       primary={
         <>
-          <Box>
+          <LayoutBox primary>
```
Comment (andy-hook) [inline src/components/Agreement/Agreement.js:48]: I've been prefixing our custom layout components with `Layout`. This makes the distinction clear vs using the same names as exist in the UI kit.

The `primary` prop is what gives the box extra padding for use in the `primary` slot. Previously the box would inherit this implicitly, which caused problems such as those [described here](https://github.com/aragon/network-dashboard/issues/19)

File: src/components/Layout/LayoutBox.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      },
+      negative: {
+        backgroundColor: '#FFFAFA',
+        borderColor: '#FF7C7C',
+      },
+      disabled: {
+        backgroundColor: theme.surfacePressed,
+        borderColor: theme.border,
+      },
+    }
```
Comment (andy-hook) [inline src/components/Layout/LayoutBox.js:24]: I'm using semantic naming here to keep this component generalised, the disputable states can then be easily mapped to use them.

File: src/components/Layout/LayoutColumns.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useLayout, GU } from '@aragon/ui'
+
+function LayoutColumns({ primary, secondary }) {
+  const { layoutName } = useLayout()
+  const oneColumn = layoutName === 'small' || layoutName === 'medium'
```
Comment (delfipolito) [inline src/components/Layout/LayoutColumns.js:7]: I know I did it like this the other time. But I just realised that here we can also use useViewport hook and do it this way:  `const oneColumn = below('medium')`. Both are fine :) whatever you prefer.

**Review by delfipolito — APPROVED** [review]

**Review by delfipolito — APPROVED** [review]
This looks awesome!! So much better 🤩 🤩 🤩

File: src/components/Layout/LayoutColumns.js (hunk 3 above)
Comment (andy-hook) [reply src/components/Layout/LayoutColumns.js:7]: That's definitely nicer, In the future i'd like to useViewport for all of these types of things, the problem is it's not set-up with our breakpoints yet and a lot of the UI kit still relies on useLayout. I was thinking we should continue to useLayout for the time being and switch it all over together :)

**Pull request merged**